library/block/block_extensions/src/testmod/java/org/quiltmc/qsl/block/extensions/test/TestBlock.java

```diff
@@ -0,0 +1,31 @@
+package org.quiltmc.qsl.block.extensions.test;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.StairsBlock;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.text.LiteralText;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.Hand;
+import net.minecraft.util.hit.BlockHitResult;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.quiltmc.qsl.block.extensions.api.QuiltBlock;
+
+public class TestBlock extends Block {
+
+    public TestBlock(Settings settings, Block... proxies) {
+        super(QuiltBlock.createProxy(settings, proxies));
+        this.setDefaultState(QuiltBlock.getProxyDefaultState(this));
+    }
+
+    @Override
+    public ActionResult onUse(BlockState state, World world, BlockPos pos, PlayerEntity player, Hand hand, BlockHitResult hit) {
+        var prop = QuiltBlock.getProperty(state, StairsBlock.HALF);
```

Comment:
This usage of getProperty doesn't sound useful?

If the stuff that was merged were merged properly, it should already include the property.